lib/robots/dor_repo/release/release_members.rb

```diff
@@ -18,12 +18,9 @@ def perform(druid)
 
           item = Dor::Release::Item.new druid: druid
 
-          case item.object_type
-          when 'collection', 'set' # this is a collection or set
-            publish_collection(druid, item)
-          else # this is not a collection of set
-            LyberCore::Log.debug "...this is a #{item.object_type}, NOOP"
-          end
+          return unless item.collection?
```

Comment:
I assume we're OK dropping the debug log entry above. It doesn't look very useful nor does it provide actionable information.